ephios/core/views/event.py

```diff
@@ -536,23 +534,18 @@ class EventCopyView(CustomPermissionRequiredMixin, SingleObjectMixin, FormView):
     permission_required = "core.add_event"
     model = Event
     template_name = "core/event_copy.html"
-    form_class = EventDuplicationForm
+    form_class = EventCopyForm
 
     def setup(self, request, *args, **kwargs):
         super().setup(request, *args, **kwargs)
         self.object = self.get_object()
 
+    def get_success_url(self):
+        return reverse("core:event_copy", kwargs={"pk": self.object.pk})
```

Comment:
When confirming the vent copy view I arrive on the event list and not at event_copy and I have no clue why :sweat_smile: